### PR TITLE
[PARTI-74] Do not generate catalog ingestion events

### PR DESCRIPTION
**Pull request by pedro-lb**
# PR Description

This PR adds the following changes:

- Removes the feature to generate catalog ingestion events, since that is no longer necessary

## ⚠️ Dependencies

- None

## 🕹 Demo

- None

# How to test

## 🧪 What should be tested?

As a reviewer, you should test:

- The catalog ingestor no longer reports ingestion events

## 📝 How to test?

Reproduce the following steps to test:

1. Perform a local ingestion
2. Check that it does not report ingestion events anymore

### CODE REVIEW

**Review by pedro-lb — COMMENTED** [review]
Self-review.

**Review by pedro-lb — COMMENTED** [review]
Self-review.

File: src/catalogIngestor/index.ts [hunk 1]
```diff
@@ -89,12 +56,6 @@ export interface Options {
    */
   apiKey: string;
 
-  /**
-   * The connection id to be used when creating ingestion events.
-   * Provided by the partner connector.
-   */
-  connectionId?: string;
```
Comment (guiquintelas) [inline src/catalogIngestor/index.ts:96]: **Suggestion**: We should probably create a linear task to update all connectors with this change.

image

**Review by guiquintelas — APPROVED** [review]
Looking good! Left a small suggestion that should not block this going forward :)

File: src/catalogIngestor/index.ts (hunk 1 above)
Comment (pedro-lb) [reply src/catalogIngestor/index.ts:96]: Agreed! Will do :D

**Pull request merged**